scripts/ci.requirements.txt

```diff
@@ -14,5 +14,4 @@ mypy >= 0.780
 # Install cryptography to avoid import-error reported by pylint.
 # What we really need is cryptography >= 35.0.0, which is only
 # available for Python >= 3.6.
-cryptography >= 35.0.0; sys_platform == 'linux' and python_version >= '3.6'
-cryptography;           sys_platform == 'linux' and python_version <  '3.6'
+cryptography # >= 35.0.0
```

Comment:
Why is the version requirement commented out?

Reply:
`tests/scripts/audit-validity-dates.py` needs `cryptography >= 35` to run correctly. But these recent versions don't support python 3.5. We don't run that script at all on the CI though. In fact, on the CI, the only thing that currently needs `cryptography` is mypy and pylint, and they're happy enough with the latest version of `cryptography` that's available for 3.5. This could use a better explanation in the comment, so I'll do that.

I'm not sure about keeping just one line for `cryptography`, or keeping the two requirements but based only on the python version. What do you think?

With no version requirement, we end up with the most recent installable version at the time the CI runs. With a version requirement, we'll end up with 35.0.0 on python ≥3.5 and an older version on Python 3.5.

Reply:
One line for `cryptography` seems fine, a more detailed comment than `>= 35.0.0` would be great!